### PR TITLE
improved php doc

### PR DESCRIPTION
**Pull request by ikwattro**
- added php image
- modified intro message
- added missing lib
- split drivers between environment usage
- added apps links using php and neo4j

### CODE REVIEW

**Comment by whatSocks** [comment]
Hey @ikwattro, could you update the adoc so that each sentence is on its own line? Makes tracking changes easier.

**Comment by ikwattro** [comment]
Hey @whatSocks not sure if I fully understand what you ask, is the last commit what you wanted ?

**Comment by whatSocks** [comment]
yes that's better thanks :D

**Comment by ikwattro** [comment]
Ok I'll change the cutter limit to a bit less to not have a single word on a line ;-)

File: language-guides/php/php.adoc [hunk 1]
```diff
@@ -25,20 +25,31 @@ toc::[]
 // tag::intro[]
 === Neo4j for PHP Developers
 
-image::http://dev.assets.neo4j.com.s3.amazonaws.com/wp-content/uploads/2014/04/php-2.png[]
+image::http://i.imgur.com/eTLVfHM.png[]
 
-Drivers for the different PHP stacks enable you to use Neo4j from your familiar environment.
+You're a PHP developer and want to start with Neo4j ? You're on the right place !
```
Comment (whatSocks) [inline language-guides/php/php.adoc]: remove spaces before punctuation marks. Technically these two should be on different lines (as they are two sentences):

You're a PHP developer and want to start with Neo4j?
You're in the right place!

File: language-guides/php/php.adoc [hunk 2]
```diff
@@ -25,20 +25,31 @@ toc::[]
...
+
+Whether you're using frameworks like Symfony and Laravel or you're developing without frameworks, you'll find an
+adequate library for you're use case.
```
Comment (whatSocks) [inline language-guides/php/php.adoc]: Should be one line and "your" not "you're":

Whether you're using frameworks like Symfony and Laravel or you're developing without frameworks, you'll find an adequate library for your use case.

File: language-guides/php/php.adoc [hunk 3]
```diff
@@ -25,20 +25,31 @@ toc::[]
...
+
+If you have any questions, you can always find a community member around on Twitter or StackOverflow by tagging your
+subjects with #neo4j-php .
 
-There’s a number of of ways to access Neo4j from PHP listed below.
 // end::intro[]
 
-Members of the PHP community have invested a lot of time and love to develop all tohese drivers, so please if you use them provide feedback to the authors.
+Members of the PHP community have invested a lot of time and love to develop all tohese drivers, so please if you use
```
Comment (whatSocks) [inline language-guides/php/php.adoc]: Hrm, looks like a previously-existing typo! "these drivers" not "tohese drivers", and should all be on one line:

Members of the PHP community have invested a lot of time and love to develop all these drivers, so if you use them please provide feedback to the authors.

File: language-guides/php/php.adoc (hunk 2 above)
Comment (ikwattro) [inline language-guides/php/php.adoc]: yes same line and yes your use case :)

File: language-guides/php/php.adoc [hunk 4]
```diff
@@ -25,20 +25,31 @@ toc::[]
...
 include::../_includes/sample_project.adoc[]
 
+=== Using Neo4j in a standalone PHP environment
+
+You develop without frameworks, these libraries are made for you :
+
 === Neo4jPHP
 
-Neo4jPHP provides an API that is both intuitive and flexible, and it takes advantage of ‘under-the -hood’ performance enhancements, such as caching and lazy-loading.
+Neo4jPHP provides an API that is both intuitive and flexible, and it takes advantage of ‘under-the -hood’ performance
```
Comment (whatSocks) [inline language-guides/php/php.adoc]: "under-the-hood"

**Comment by ikwattro** [comment]
better ?

**Comment by whatSocks** [comment]
Sure, I'll merge fix the little english mistakes in a little bit.

**Comment by ikwattro** [comment]
Ok cool thanks ;-)

File: language-guides/php/php.adoc [hunk 5]
```diff
@@ -25,20 +25,32 @@ toc::[]
...
+You're on the right place!
+
+Whether you're using frameworks like Symfony and Laravel or you're developing without frameworks, you'll
+find an adequate library for your use case.
+
+If you have any questions, you can always find a community member around on Twitter or StackOverflow by
+tagging your subjects with #neo4j-php .
 
-There’s a number of of ways to access Neo4j from PHP listed below.
 // end::intro[]
```
Comment (jexp) [inline language-guides/php/php.adoc:39]: move //end::intro[] above the stackoverflow sentence

File: language-guides/php/php.adoc [hunk 6]
```diff
@@ -92,3 +121,50 @@ A Neo4j Laravel 4.1 OGM Provider.
 
 * https://github.com/lrezek/Neo4Laravel[Neo4Laravel on GitHub]
 * Author: https://github.com/lrezek[Lukas Rezek]
+
+''''
+
+=== Applications built with PHP and Neo4j
```
Comment (jexp) [inline language-guides/php/php.adoc:127]: Put Applications behind all drivers.

File: language-guides/php/php.adoc [hunk 7]
```diff
@@ -92,3 +121,50 @@ A Neo4j Laravel 4.1 OGM Provider.
...
+
+=== Omnomhub
```
Comment (jexp) [inline language-guides/php/php.adoc:129]: Level 4 for Omnohub

File: language-guides/php/php.adoc [hunk 8]
```diff
@@ -92,3 +121,50 @@ A Neo4j Laravel 4.1 OGM Provider.
...
+
+Like for Github, but for recipes ! 
+
+* Author: https://twitter.com/michellesanver[Michelle Sanver]
+
+link:https://github.com/Omnomhub/omnomhub[https://github.com/Omnomhub/omnomhub]
+
+=== Thesaurus Manager
```
Comment (jexp) [inline language-guides/php/php.adoc:137]: Level 4 for Thesaurus Manager

File: language-guides/php/php.adoc [hunk 9]
```diff
@@ -92,3 +121,50 @@ A Neo4j Laravel 4.1 OGM Provider.
...
+
+If you want to built your own thesaurus, this app is made for you
+
+* Author: https://github.com/hasantayyar[Hasan Tayyar]
+
+link:http://okulbilisim.github.io/thesaurus-manager/[http://okulbilisim.github.io/thesaurus-manager/]
```
Comment (jexp) [inline language-guides/php/php.adoc:143]: Use List item with `*` for the Link and use a text like `Thesaurus Manager on GitHub` for the link text

File: language-guides/php/php.adoc [hunk 10]
```diff
@@ -92,3 +121,50 @@ A Neo4j Laravel 4.1 OGM Provider.
...
+
+=== Applications built with PHP and Neo4j
+
+=== Omnomhub
+
+Like for Github, but for recipes ! 
+
+* Author: https://twitter.com/michellesanver[Michelle Sanver]
+
+link:https://github.com/Omnomhub/omnomhub[https://github.com/Omnomhub/omnomhub]
```
Comment (jexp) [inline language-guides/php/php.adoc:135]: Use List item with `*` for the Link and use a text like `Omnomhub on GitHub` for the link text

File: language-guides/php/php.adoc [hunk 11]
```diff
@@ -58,33 +85,35 @@ An actively developed PHP HttpClient for the Neo4j REST and Cypher API with Mult
...
 
-* http://github.com/endyjasmi/cypher[PHP Cypher on GitHub]
-* Author: https://github.com/endyjasmi[Endy Jasmi]
+https://github.com/klaussilveira/neo4j-ogm-bundle[https://github.com/klaussilveira/neo4j-ogm-bundle]
+
+''''
+
+=== You use the Laravel framework
 
 ==== NeoEloquent
```
Comment (jexp) [inline language-guides/php/php.adoc:111]: perhaps include NeoEloquent for Laravel into the header, and remove the extra "empty" header "You use the Laravel framework"?

File: language-guides/php/php.adoc [hunk 12]
```diff
@@ -25,20 +25,32 @@ toc::[]
...
-There’s a number of of ways to access Neo4j from PHP listed below.
 // end::intro[]
 
-Members of the PHP community have invested a lot of time and love to develop all tohese drivers, so please if you use them provide feedback to the authors.
+Members of the PHP community have invested a lot of time and love to develop all tohese drivers, so please
+if you use them provide feedback to the authors.
 
 include::../_includes/sample_project.adoc[]
 
+=== Using Neo4j in a standalone PHP environment
```
Comment (jexp) [inline language-guides/php/php.adoc:46]: Perhaps we can change this section into talking the different ways (standalone, symfony, laravel) of using a driver within PHP and then linking to each of the drivers? Not sure.

File: language-guides/php/php.adoc [hunk 13]
```diff
@@ -25,20 +25,32 @@ toc::[]
 // tag::intro[]
 === Neo4j for PHP Developers
 
-image::http://dev.assets.neo4j.com.s3.amazonaws.com/wp-content/uploads/2014/04/php-2.png[]
+image::http://i.imgur.com/eTLVfHM.png[]
```
Comment (jexp) [inline language-guides/php/php.adoc:28]: Please use this image: http://dev.assets.neo4j.com.s3.amazonaws.com/wp-content/uploads/neo4j-loves-php.png

**Comment by whatSocks** [comment]
@ikwattro I cleaned up the file but did so before @jexp added his comments, so you'll have to pull again before implementing his changes.

**Comment by ikwattro** [comment]
Yes I'll rebase on the gh-pages branch and open a new PR

**Pull request merged**